source/pki_utils.c

```diff
@@ -226,3 +226,76 @@ int aws_read_and_decode_pem_file_to_buffer_list(
 
     return AWS_OP_SUCCESS;
 }
+
+int aws_decode_pem_to_der_buf(
+    struct aws_allocator *alloc,
+    const struct aws_byte_cursor *pem_cursor,
+    struct aws_byte_buf *der_buf) {
+    aws_byte_buf_clean_up(der_buf);

```

Comment:
can user pass in buffer thats already initialized? should it just be zero out instead of clean up?

Reply:
The function is deprecated.